org.OpenT2T.Sample.SuperPopular.Thermostat/Wink Thermostat/v2/org.OpenT2T.Sample.SuperPopular.Thermostat-2.0.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+    <description>OpenT2T SuperPopular Thermostat interface</description>
+
+    <property name="PowerStatus" type="s" access="readwrite">
+      <description>Get/Set power status. (On, Off)</description>
+    </property>
+
```

Comment:
it is kind of wierd. Both are same. In case of power on, a default mode (heat-cool in case of Nest). Specific mode (heat/cool etc...) can be set separately.
Wink provides different property (powered) to turn on/off. But it ultimately sets hvac_mode. Nest has only hvac_mode property, which we are using in both setting mode and power on/off.

Regarding Boolean: I would make sense. I couldn't think of a proper property name. If it is 'Powered', methods will be getPowered and setPowered, which do not sound good.

---
In reply to: [73765703](https://github.com/openT2T/translators/pull/37#discussion_r73765703) [](ancestors = 73765703)